src/uv.h

```diff
@@ -318,7 +318,8 @@ int UvPrepare(struct uv *uv,
  * and then remove it immediately. */
 void UvPrepareClose(struct uv *uv);
 
-/* Implementation of raft_io->append.*/
+/* Implementation of raft_io->append. All the raft_buffers of the raft_entry structs
+ * in the entries array are required to have a len that is a multiple of 8. */
```

Comment:
Do we want the alignment requirement to be part of the contract of raft_io->append more generally?

Reply:
I don't think so, it's an implementation detail of the uv backend imo.